cinema/urls.py

```diff
@@ -1 +1,29 @@
-# write urls here
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from cinema.views import (
+    GenreViewSet,
+    MovieViewSet,
+    ActorViewSet,
+    MovieSessionViewSet,
+    CinemaHallViewSet,
+)
+
+app_name = "cinema"
+router = DefaultRouter()
+router.register("genres", viewset=GenreViewSet, basename="genres")
+router.register("movies", viewset=MovieViewSet, basename="movies")
+router.register("actors", viewset=ActorViewSet, basename="actors")
+router.register(
+    "cinema_halls",
+    viewset=CinemaHallViewSet,
+    basename="cinema_halls"
+)
```

Comment:
The router URLs are included twice in `urlpatterns`, once at the root and once under `app/cinema/`. This redundancy could lead to duplicate routes. Consider removing one of these path inclusions to avoid potential confusion.